test/sql/test_conf/T/test_configure

```diff
@@ -0,0 +1,32 @@
+-- name: test_configure
+admin set frontend config ("thrift_max_message_size"="10485760");
```

Comment:
shall reset these configurations after test done, and mark the test case as @sequential, so it won't break other cases running in parallel.

The same to the be_configs.